### DISCUSSION TITLE
PRs in forked repositories

### DISCUSSION THREAD

**Discussion by njzjz** [renovatebot/renovate#21138] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

In the past, renovate didn't create PRs in forked repositories.

### What you tried so far.

Since 2 hours ago, renovate has created a lot of PRs in my forked repositories. I am wondering if there is anything changed and how I can disable it.

![Screenshot from 2023-03-24 16-36-31](https://user-images.githubusercontent.com/9496702/227633752-4f20a434-de55-43d1-8674-aa2f78788a7a.png)

### Relevant debug logs

I don't find any logs in renovate's app dashboard for these repositories.

**Comment by rarkins** [comment]
Have you installed into "All repositories" or "Select repositories"?

**Reply by njzjz** [reply]
I installed renovate into "All repositories". I will try "Select repositories". Thanks.

**Comment by rarkins** [comment]
Image doesn't seem to work for me from mobile

**Reply by viceice** [reply]
github tls certificate was expired 😅
